programs/management/commands/add_counties.py

```diff
@@ -0,0 +1,31 @@
+from django.core.management.base import BaseCommand
+from programs.models import County
+from screener.models import WhiteLabel
+from configuration.models import Configuration
+from django.core.exceptions import ObjectDoesNotExist
+
+
+class Command(BaseCommand):
+    help = "Adds counties"
+
+    def add_arguments(self, parser):
+        parser.add_argument("white_label", type=str, help="The state code for the white label")
+
+    def handle(self, *args, **options):
+        try:
+            white_label = WhiteLabel.objects.get(code=options["white_label"])
+        except ObjectDoesNotExist:
+            self.stdout.write(self.style.WARNING(f'White label for "{options["white_label"]}" is not in the database'))
```

Comment:
We want to stop the script if there is an error.
```suggestion
            self.stdout.write(self.style.WARNING(f'White label for "{options["white_label"]}" is not in the database'))
            return
```